allel/util.py

```diff
@@ -145,6 +146,36 @@ def ensure_square(dist):
     return dist
 
 
+def mask_inaccessible(is_accessible, ac, pos):
+    """ This function removes any positions that are not accessible from the
+    allele counts Array and the vector of variant positions.
+
+    Parameters
+    ----------
+    is_accessible : array_like, bool, shape (len(contig),), optional
+        Boolean array indicating accessibility status for all positions in the
+        chromosome/contig.
+    ac : array_like, int, shape (n_variants, n_alleles)
+        Allele counts array.
+    pos : array_like, int, shape (n_items,)
+        Variant positions, using 1-based coordinates, in ascending order.
+
+    Returns
+    -------
+    ac : array_like, int, shape (n_variants, n_alleles)
+        Masked allele counts array.
+    pos : array_like, int, shape (n_items,)
+        Masked variant positions, using 1-based coordinates, in ascending order.
```

Comment:
Maybe instead of "masked" say for instance "Allele counts array corresponding to only the accessible sites in the original allele counts array."